src/main/java/net/dries007/tfc/common/recipes/IItemRecipe.java

```diff
@@ -0,0 +1,12 @@
+package net.dries007.tfc.common.recipes;
+
+import net.minecraft.world.World;
+
+public interface IItemRecipe extends ISimpleRecipe<ItemStackRecipeWrapper>
```

Comment:
This seems pointless? `matches` will need to be overridden by subclasses always.